M/MicroMamba/Versions.toml

```diff
@@ -27,6 +27,3 @@ git-tree-sha1 = "18ae2d81035c717f9b0d92c809575266dfe73cc9"
 
 ["0.1.9"]
 git-tree-sha1 = "b49f11c4f9272fcbd66816ab8c3128d39982966c"
-
-["0.1.10"]
-git-tree-sha1 = "539c0d9ab5cc3bca6c2bfdfbf2b050d99390bf52"
```

Comment:
You have to add yanked = true after the version, not just removing these lines which would break existing code